src/create_disk.sh

```diff
@@ -87,6 +88,20 @@ do
 done
 ostree admin deploy "${deploy_ref}" --sysroot rootfs --os "$os_name" $kargsargs
 
+# This will allow us to track the version that an install
+# originally used; if we later need to understand something
+# like "exactly what mkfs.xfs version was used" we can do
+# that via looking at the upstream build and finding the
+# build logs for it, getting the coreos-assembler version,
+# and getting the `rpm -qa` from that.
+ostree_commit=$(ostree --repo="${ostree}" rev-parse "${ref}")
+cat > rootfs/.coreos-aleph-version.json << EOF
+{
+	"build": "${buildid}",
+	"ostree-commit": "${ostree_commit}"
```

Comment:
Let's also add `ostree-ref` here? Since that also encodes the stream they started on (esp. since build IDs are not unique per stream).